integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/rdd/SecondaryIndexCreator.scala

```diff
@@ -398,6 +404,28 @@ object SecondaryIndexCreator {
           secondaryIndexModel.sqlContext.sparkSession,
           carbonLoadModelForMergeDataFiles.getFactTimeStamp,
           rebuiltSegments)
+
+        if (isInsertOverwrite) {
+          var staleSegmentsList = new ListBuffer[String]()
```

Comment:
`staleSegmentsList ` name depicts wrong info, better to rename variable to `overriddenSegments`

Reply:
removed